### DISCUSSION TITLE
Maui Full Screen (Kisok Mode) Android App Support

### DISCUSSION THREAD

**Discussion by sujithsmenon** [dotnet/maui#13045] [Q&A] [answered]
Hello,
I am new to .Net Maui.
i am a asp.net, asp.net core programmer, we are developing only web application. But one of our client special requirement we need to develop mobile application, so i decided to develop in .Net Maui because of C#.  developed the app they required but the problem is now that they need the app in kiosk mode becuase when they give the mobile to inspection team, the app should always required them as kiosk mod.
can any one suggest me how to show a login screen when the user press the home & back keys in android device, if super admin login ok user can exit from the kisok mode.

**Comment by acaliaro** [comment, marked as answer]
it's not easy to manage the kiosk for an app. Usually this functionality is given by MDM software. Here a link: https://developer.android.com/work/dpc/dedicated-devices/lock-task-mode

You can try creating a "launcher" type app, which replaces the Android launcher. It should just be a manifest side change

an example of how to modify the manifest can be found here:
https://steemit.com/utopian-io/@ideba/how-to-build-a-custom-android-launcher-and-home-screen-application-part-1

**Reply by sujithsmenon** [reply]
thank  you @acaliaro .
